lua/lvim/plugins.lua

```diff
@@ -177,6 +177,15 @@ return {
     end,
     disable = not lvim.builtin.dashboard.active,
   },
+  -- alpha
+  {
+    "goolord/alpha-nvim",
+    event = "BufWinEnter",
+    config = function()
+      require("lvim.core.alpha").setup()
+    end,
+    -- disable = not lvim.builtin.alpha.active,
```

Comment:
i think we can still have
```lua
disable = not lvim.builtin.dashboard.active
```
or sth here

Reply:
Yeah I just need to make sure it's initialized before calling it.